src/fastapi_poe/base.py

```diff
@@ -629,6 +645,263 @@ def make_prompt_author_role_alternated(
 
         return new_messages
 
+    async def capture_cost(
+        self,
+        request: QueryRequest,
+        amounts: Union[List[CostItem], CostItem],
+        access_key: Optional[str] = None,
+        base_url: str = "https://api.poe.com/",
+    ) -> None:
+        """
+
+        Used to capture a cost for monetized creators.
```

Comment:
imo this can be more descriptive on what exactly "capture" is doing as creators may not be familiar with the capture/auth framework. could link to a doc for example.

Reply:
Good call, added the link to the doc that should be updated with more information.